src/main/java/teammates/ui/controller/AdminActivityLogPageAction.java

```diff
@@ -64,30 +64,33 @@ protected ActionResult execute() throws EntityDoesNotExistException{
 //      should be shown. Use "?all=true" in URL to show all logs. This will keep showing all
 //      logs despite any action or change in the page unless the the page is reloaded with "?all=false" 
 //      or simply reloaded with this parameter omitted.
-        data.ifShowAll = getRequestParamAsBoolean("all");
+        boolean ifShowAll = getRequestParamAsBoolean("all");
         
```

Comment:
Should this be isShowAll instead of ifShowAll? You can create an issue for this.